Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -2841,29 +2844,33 @@ RefPtr<CSSStyleValue> Parser::parse_rect_value(TokenStream<ComponentValue>& toke
 }
 
 // https://www.w3.org/TR/css-color-4/#typedef-hue
-RefPtr<CSSStyleValue> Parser::parse_hue_value(TokenStream<ComponentValue>& tokens)
+RefPtr<CSSStyleValue> Parser::parse_hue_none_value(TokenStream<ComponentValue>& tokens)
 {
-    // <hue> = <number> | <angle>
+    // Parses [<hue> | none]
+    //   <hue> = <number> | <angle>
     if (auto number = parse_number_value(tokens))
         return number;
     if (auto angle = parse_angle_value(tokens))
         return angle;
+    if (auto keyword = parse_keyword_value(tokens); keyword && keyword->as_keyword().keyword() == Keyword::None)
```

Comment:
Same point about consuming tokens and using a transaction.

Reply:
Fixed